### PR TITLE
adds KUBERNETES execTemplates for U1404

### PR DESCRIPTION
**Pull request by Riturajcse**
https://github.com/Shippable/execTemplates/issues/180

### CODE REVIEW

File: Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    exit 1
+  fi
+
+  _log_success "Successfully checked params"
+}
+
+init_scope_configure() {
+  _log_msg "Initializing scope configure"
+
+  mkdir ~/.kube
```
Comment (shrivara) [inline Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh:40]: Why is this line using a relative path and next line an absolute?

File: Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  Usage:
+    $SCRIPT_NAME <resource_name> [scopes]
+  "
+}
+
+check_params() {
+  _log_msg "Checking params"
+
+  KUBERNETES_INT="$(shipctl get_integration_resource "$RESOURCE_NAME")"
+  KUBERNETES_KUBECONFIGFILE="$( echo "$KUBERNETES_INT" | jq -r '.kubeConfigContent' )"
```
Comment (Bharath92) [inline Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh]: is this `kubeConfigContent` or `masterKubeConfigContent`?

**Review by Bharath92 — CHANGES_REQUESTED** [review]

File: Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh (hunk 2 above)
Comment (Riturajcse) [reply Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh]: This is `kubeConfigContent`.

File: Ubuntu_14.04/resources/cliConfig/kubernetesConfig/init.sh [hunk 3]
```diff
@@ -38,7 +38,7 @@ init_scope_configure() {
   _log_msg "Initializing scope configure"
 
   mkdir ~/.kube
-  echo "$KUBERNETES_KUBECONFIGFILE" > /root/.kube/config
+  echo "$KUBERNETES_KUBECONFIGFILE" > /.kube/config
```
Comment (Bharath92) [inline Ubuntu_14.04/resources/cliConfig/kubernetesConfig/init.sh]: missing ~

**Review by Bharath92 — CHANGES_REQUESTED** [review]

File: Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+check_params() {
+  _log_msg "Checking params"
+
+  KUBERNETES_INT="$(shipctl get_integration_resource "$RESOURCE_NAME")"
+  KUBERNETES_KUBECONFIGFILE="$( echo "$KUBERNETES_INT" | jq -r '.masterKubeConfigContent' )"
+
+  if _is_empty "$KUBERNETES_KUBECONFIGFILE"; then
+    _log_err "Missing 'kubeConfigFile' value in $RESOURCE_NAME's integration."
```
Comment (shrivara) [inline Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh]: Missing `masterKubeConfigContent`...

**Review by shrivara — APPROVED** [review]

**Review by Bharath92 — APPROVED** [review]

**Pull request merged**